reana_commons/publisher.py

```diff
@@ -163,7 +164,13 @@ def __init__(self, **kwargs):
         )
 
     def publish_workflow_submission(
-        self, user_id, workflow_id_or_name, parameters, priority=0, min_job_memory=0,
+        self,
+        user_id,
+        workflow_id_or_name,
+        parameters,
+        priority=0,
+        min_job_memory=0,
+        retry_count: Optional[int] = None,
```

Comment:
Minor: alternative approach would be to always send `retry_count` value initialized as 0 by default (as `priority` and `min_job_memory`)